### ISSUE TITLE
BatchSize

### ISSUE THREAD

**Issue by bluesky314** [PeterL1n/BackgroundMattingV2#123]
F, B, Alpha Matting(https://arxiv.org/abs/2003.07711) did experiments to show that batch size of 1 greatly outperforms larger batch sizes for matting. According to them one can achieve very different results if larger batch size is used. Can you take about your batch size strategy. Did you try this or do you have any observation about this result of theirs?

**Comment by PeterL1n** [comment]
We don't use GroupNorm so batch size = 1 won't make sense. We simply use batch norm so that we can use other's pre-trained weights.

**Comment by bluesky314** [comment]
Is it recommended to train with a large batch size of 32? Wondering if there is any plus side to training with smaller batch sizes for such dense prediction tasks

**Comment by PeterL1n** [comment]
Our batch size is specified in the paper. No comment for the effects of using other batch sizes.
